core/src/main/scala/knobs/Parser.scala

```diff
@@ -38,22 +38,22 @@ object ConfigParser {
   }
 
   /** Top-level parser of configuration files */
-  lazy val topLevel: Parser[List[Directive]] = "configuration" |: {
-    directives << skipLWS << realEOF
-  }
+  lazy val topLevel: Parser[List[Directive]] = topLevel(withImport = true)
```

Comment:
I think this can be fixed in two vals (`sansImport` and `directives`) without any breaking changes.  While I agree that version bumps are relatively cheaper in this sort of library, I'd still prefer to avoid churn where it's easy.